scripts/azureml-assets/azureml/assets/validate_tree.py

```diff
@@ -0,0 +1,65 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+"""Validate source tree files."""
+
+import argparse
+import sys
+from pathlib import Path
+from typing import List
+
+import azureml.assets.util as util
+from azureml.assets.util import logger
+
+BYTES_IN_A_MEGABYTE = 1024 ** 2
+
+
+def validate_tree(input_dirs: List[Path]):
+    """Validate source tree files.
+
+    Args:
+        input_dirs (List[Path]): List of directories to search for assets.
+
+    Returns:
+        bool: True if files were successfully validated, otherwise False.
+    """
+    error_count = 0
+
+    for file in util.find_files(input_dirs, "*"):
+
+        # Check that every spec.yaml has asset.yaml next to it
+        if file.name == "spec.yaml":
+            asset_config_file = file.parent / "asset.yaml"
```

Comment:
Please use [this constant](https://github.com/Azure/azureml-assets/blob/ed2b533f1d4970621932dae773aff82d08261e3d/scripts/azureml-assets/azureml/assets/config.py#L110) instead of hard-coding asset.yaml. Also applies to the error message two lines down.